hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHA.java

```diff
@@ -122,6 +124,8 @@ public void init() throws Exception {
     clusterId = UUID.randomUUID().toString();
     scmId = UUID.randomUUID().toString();
     conf.setBoolean(OZONE_ACL_ENABLED, true);
+    conf.set(OzoneConfigKeys.OZONE_ADMINISTRATORS,
```

Comment:
Why is this needed?

Reply:
Because Ozone acls are enabled, and for volume creation the acl logic is as below. For testing purpose gave "*". 
```
      // check Acl
      if (ozoneManager.getAclsEnabled()) {
        if (!ozAdmins.contains(OZONE_ADMINISTRATORS_WILDCARD) &&
            !ozAdmins.contains(getUserInfo().getUserName())) {
          throw new OMException("Only admin users are authorized to create " +
              "Ozone volumes. User: " + getUserInfo().getUserName(),
              OMException.ResultCodes.PERMISSION_DENIED);
        }
      }
```